test/integration/mtime-update.js

```diff
@@ -53,13 +53,9 @@ describe('Update only a file mtime', () => {
     // actually change the file
     await fse.appendFile(localPath, ' appended')
     const stats = await fse.stat(localPath)
+    statAfter.mtime.toISOString().should.not.equal(stats.mtime.toISOString())
 
-    await helpers.local.simulateEvents([{
-      type: 'change',
-      path: 'file',
-      stats: stats
-    }])
-
+    await helpers.local.scan()
```

Comment:
This probably ends up with a different event & test.
Better check watcher type and simulate appropriate event first to make sure we don't introduce regressions.
We can still refactor this afterwards.